puppet/zulip/manifests/postgres_common.pp

```diff
@@ -44,6 +47,11 @@
       exec {'pip3_deps':
         command => 'python3 -m pip install pytz python-dateutil'
       }
+      # Create ssl-cert group and allows it to read /etc/pki/tls/private
+      exec {'ssl-cert_group':
+        command => 'groupadd -f ssl-cert && chgrp -R ssl-cert /etc/pki/tls/private'
```

Comment:
This will get run on every puppet run, which is probably bad.

Reply:
You can declare a user/group a better way; see `base.pp`.

Reply:
Fixed.